src/inspector_socket.cc

```diff
@@ -478,9 +483,20 @@ class HttpHandler : public ProtocolHandler {
   }
 
   void OnData(std::vector<char>* data) override {
+    parser_errno_t err;
+#ifdef NODE_EXPERIMENTAL_HTTP
+    err = llhttp_execute(&parser_, data->data(), data->size());
+
+    if (err == HPE_PAUSED_UPGRADE) {
```

Comment:
Can you explain the necessity for this edge case @indutny? On the surface it looks like something that shouldn't be handled here; or is it simply enabling a hook for specialised use case that we don't need?

Reply:
Yes, you're right. It is just a special case that we have no use for.